apps/test/unit/netsim/NetSimRouterLogModal.js

```diff
@@ -123,6 +125,20 @@ describe("NetSimRouterLogModal", function () {
       modal.setRouter(null); // Normally netsim.js does this
       assert.equal('none', modal.currentTrafficFilter_);
     });
+
+    it("can render the dropdown with no local address", function () {
+      expect(() => {
+        modal.onShow_();
+      }).not.to.throw();
+    });
+
+    it("can render the dropdown with a local address", function () {
+      modal.onShardChange(testShard, localNode);
+      localNode.connectToRouter(router);
+      expect(() => {
+        modal.onShow_();
+      }).not.to.throw();
```

Comment:
Confirmed that this test fails without my fix:

![screenshot from 2016-07-17 15-59-34](https://cloud.githubusercontent.com/assets/1615761/16903611/2e2d0f42-4c38-11e6-8616-212217196b15.png)